modules/Activities/activities_manage_addProcess.php

```diff
@@ -26,36 +26,36 @@
     header("Location: {$URL}");
 } else {
     //Proceed!
-    $name = $_POST['name'];
-    $provider = $_POST['provider'];
-    $active = $_POST['active'];
-    $registration = $_POST['registration'];
-    $dateType = $_POST['dateType'];
+    $name = $_POST['name'] ?? '';
+    $provider = $_POST['provider'] ?? '';
+    $active = $_POST['active'] ?? '';
+    $registration = $_POST['registration'] ?? '';
+    $dateType = $_POST['dateType'] ?? '';
     if ($dateType == 'Term') {
-        $gibbonSchoolYearTermIDList = isset($_POST['gibbonSchoolYearTermIDList'])? $_POST['gibbonSchoolYearTermIDList'] : array();
+        $gibbonSchoolYearTermIDList =  $_POST['gibbonSchoolYearTermIDList'] ?? array();
         $gibbonSchoolYearTermIDList = implode(',', $gibbonSchoolYearTermIDList);
     } elseif ($dateType == 'Date') {
-        $listingStart = dateConvert($guid, $_POST['listingStart']);
-        $listingEnd = dateConvert($guid, $_POST['listingEnd']);
-        $programStart = dateConvert($guid, $_POST['programStart']);
-        $programEnd = dateConvert($guid, $_POST['programEnd']);
+        $listingStart = dateConvert($guid, $_POST['listingStart']) ?? '';
+        $listingEnd = dateConvert($guid, $_POST['listingEnd']) ?? '';
+        $programStart = dateConvert($guid, $_POST['programStart']) ?? '';
+        $programEnd = dateConvert($guid, $_POST['programEnd']) ?? '';
```

Comment:
Hi Harry. Unfortunately because these are using the dateConvert function they don't work with a null coalesce. They'll need to use a generic isset check in a ternary instead. Here's an example of the first line, do you want to try your hand at updating the rest:
`$listingStart = isset($_POST['listingStart']) ? dateConvert($guid, $_POST['listingStart']) : '';`